hawc/apps/assessment/templates/assessment/assessment_detail.html

```diff
@@ -138,6 +138,11 @@ <h1>{{object}}
           <td>{{object.public}}</td>
         </tr>
 
+        <tr>
+          <th>Public Training Data</th>
```

Comment:
Change to `Public training data`; I know it's inconsistent in HAWC, but I try to capitalize the first word only